backend/modules/models.py

```diff
@@ -0,0 +1,50 @@
+from uuid import uuid4
+from cassandra.cqlengine import columns
+from django_cassandra_engine.models import DjangoCassandraModel
+
+
+class Answers(DjangoCassandraModel):
```

Comment:
answers already defined in the "question" module -> should remove the one in the questions app